scripts/check-public-api.ts

```diff
@@ -8,12 +8,16 @@ import {
   readCompilerOptions,
   transpileDirectory
 } from '@sap-cloud-sdk/generator-common/internal';
+
 const { readFile, lstat, readdir } = promises;
+
 const logger = createLogger('check-public-api');
+
 const pathToTsConfigRoot = join(__dirname, '../tsconfig.json');
 const pathRootNodeModules = resolve(__dirname, '../node_modules');
 export const regexExportedIndex = /\{([\w,]+)\}/g;
```

Comment:
Perhaps:  `export const regexExportedIndex = /\{([\w,]+)\}from'\./g;`
Sorry if my comment wasn't clear before. The regex `/\{([\w,]+)\}from'\./g` itself was correct as it ignores the public re-exports(from '@test-package/') and only considers from './some-file'.

Just wanted to used this variable since its used in the tests also.

Reply:
Yes, this wasn't on purpose. Should be ok now.